servo/runner.py

```diff
@@ -147,6 +147,7 @@ async def exec_command(self) -> servo.api.Status:
             except servo.errors.EventError as error:
                 self.logger.info(f"Measurement failed: {error}")
                 param = servo.api.Status.from_error(error)
+                self.logger.info(f"Responding with status {param.status}")
```

Comment:
I see that we are using the info log level above but does it make more sense to utilize `error`? Not sure what the output looks like

Reply:
Logging with error changes the color of the log text from white to red and uses `ERROR` after the timestamp instead of `INFO`. I agree that since the logic is error handling, error is a more appropriate logging method and I've updated the implementation accordingly